chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd

```diff
@@ -0,0 +1,1088 @@
+---
+title: "ScPoEconometrics"
+subtitle: "Confidence Intervals and Hypothesis Testing"
+author: "Florian Oswald, Gustave Kenedi and Pierre Villedieu"
+date: "SciencesPo Paris </br> `r Sys.Date()`"
+output:
+  xaringan::moon_reader:
+    lib_dir: libs
+    css: [default, "../css/scpo.css", "../css/scpo-fonts.css"]
+    nature:
+      beforeInit: ["../js/ru_xaringan.js"]
+      highlightStyle: github
+      highlightLines: true
+      countIncrementalSlides: false
+      ratio: "16:9"
+    includes:
+      in_header: "../libs/partials/header.html"
+---
+
+layout: true
+
+<div class="my-footer"><img src="../img/logo/ScPo-shield.png" style="height: 60px;"/></div> 
+
+---
+
+```{r setup, include=FALSE,warning=FALSE,message=FALSE}
+options(htmltools.dir.version = FALSE)
+knitr::opts_chunk$set(
+  message = FALSE,
+  warning = FALSE,
+  dev = "svg",
+  cache = TRUE,
+  fig.align = "center"
+  #fig.width = 11,
+  #fig.height = 5
+)
+
+# define vars
+om = par("mar")
+lowtop = c(om[1],om[2],0.1,om[4])
+
+library(magrittr)
```

Comment:
i think it's good practice to show people which packages they need to load on a slide such that the code works.

Reply:
i agree, we've inserted in the slides the packages they need `moderndive` and `infer`. the only one missing is `tidyverse` but i would assume that students know that it should be loaded by now? i'd be happy to add it to slide 7!

regarding code, we do not show the code on how to construct the null distribution because we judged it wasn't essential to understanding the intuition. but we could add it somewhere if you would like.

Reply:
just add all libraries you load, so there is no ambiguity what needs to be done to reproduce the stuff.